docs/topics/feed-exports.rst

```diff
@@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
 
 .. setting:: FEED_EXPORT_FIELDS
 
+FEED_EXPORT_ENCODING
+--------------------
+
+The encoding to be used for the feed. Defaults to UTF-8.
```

Comment:
This is not correct for JSON(lines), the default encoding is `\uXXXX` escape sequences.
A special case indeed, but worth a mention I believe.